earthfile2llb/listener.go

```diff
@@ -370,6 +371,10 @@ func (l *listener) ExitRunStmt(c *parser.RunStmtContext) {
 			l.err = fmt.Errorf("the --privileged flag has no effect when used with the LOCALLY directive: %s", c.GetText())
 			return
 		}
+		if *noCache {
```

Comment:
Didn't make any special effort here. Of course its not cached? Any bad or invalid assumptions I made and need to check?

Reply:
This seems like a good decision.